.NET/Microsoft.Recognizers.Definitions.Common/Spanish/NumbersDefinitions.cs

```diff
@@ -58,7 +58,7 @@ public static class NumbersDefinitions
       public static readonly string SufixRoundOrdinalRegex = $@"(({AllIntRegex})({SimpleRoundOrdinalRegex}))";
       public static readonly string ComplexRoundOrdinalRegex = $@"((({SufixRoundOrdinalRegex}(\s)?)?{ComplexOrdinalRegex})|{SufixRoundOrdinalRegex})";
       public static readonly string AllOrdinalRegex = $@"{ComplexOrdinalRegex}|{SimpleRoundOrdinalRegex}|{ComplexRoundOrdinalRegex}";
-      public const string OrdinalSuffixRegex = @"(?<=\b)(\d*((1r[oa]|2d[oa]|3r[oa]|4t[oa]|5t[oa]|6t[oa]|7m[oa]|8v[oa]|9n[oa]|0m[oa]|11[vm][oa]|12[vm][oa])|(1|2|3|4|5|6|7|8|9|0)[ºª]))(?=\b)";
+      public const string OrdinalSuffixRegex = @"(?<=\b)(\d*((1(er|r[oa])|2d[oa]|3r[oa]|4t[oa]|5t[oa]|6t[oa]|7m[oa]|8v[oa]|9n[oa]|0m[oa]|11[vm][oa]|12[vm][oa])|\d\.?[º°ª]))(?=\b)";

```

Comment:
Should we really add the degree symbol ('°') here? Do people use it instead of the correct superscript o?

Reply:
I included it since the two symbols are similar and could be confused. Spanish keyboards have the ordinal indicator, but other keyboard layouts (German, French, Italian...) have only the degree symbol. 
But it can be removed from the regex if it is problematic.

Reply:
Please remove it in the next PR.

Reply:
ok, removed.